src/status_im/contexts/keycard/change_pin/events.cljs

```diff
@@ -63,11 +66,10 @@
 
 (rf/reg-event-fx :keycard/change-pin.verify-current-pin-and-continue
  (fn [{:keys [db]}]
-   (let [{:keys [current-pin new-pin]} (get-in db [:keycard :change-pin])
-         unmasked-current-pin          (security/safe-unmask-data current-pin)
-         unmasked-new-pin              (security/safe-unmask-data new-pin)]
+   (let [key-uid         (get-in db [:profile/profile :key-uid])
```

Comment:
it's better to use instance-uid here, because it might be backup card for example with the same key. And compare instance-uid in all next scans

Reply:
Thank you @flexsurfer for your reviewer.

> it's better to use instance-uid here, because it might be backup card for example with the same key.

So how we can change pin for backup card?

Reply:
> it's better to use instance-uid here,

Sorry I am confused, how to use instance-uid here? `:keycard/connect` don't have this param

Reply:
hi @flexsurfer, Is this PR ready for testing?